x-pack/test/functional_enterprise_search/page_objects/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { pageObjects as basePageObjects } from '../../functional/page_objects';
```

Comment:
(This is not necessarily a change request since we appear to be reusing existing architecture, just me thinking out loud)

Maybe I'm missing something... but why put our page objects in `x-pack/test/functional/page_objects` when we could just put them in `x-pack/test/functional_enterprise_search/page_objects` instead?

Reply:
I think I mean to delete that one from `pack/test/functional/page_objects`, I thought I did anyway. Let me clean that up, because I agree with you.

Reply:
Done: https://github.com/constancecchen/kibana/pull/6